src/MultiVariablePolynomialTest.java

```diff
@@ -0,0 +1,29 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class MultiVariablePolynomialTest
+{
+	MultiVariablePolynomial poly;
+	@Test
+	public void test_poly()
+	{
+		String polynomial="3x2y4-5z2+x-y+20";
```

Comment:
1. Update the method name so that it describe what scenario you are testing inside it.
1. Correct indentation here

Reply:
done